content/docs/1.7.0/advanced-resources/os-distro-specific/container-optimized-os-support.md

```diff
@@ -0,0 +1,67 @@
+---
+title:  Container-Optimized OS Support
+weight: 5
+---
+
+## Requirements
+
+> **Note:**
+> Longhorn currently supports Container-Optimized OS only when used as the base image for Google Kubernetes Engine (GKE), which includes a pre-configured Kubernetes environment. The following information may not apply to manually created Kubernetes environments, including Kubernetes provisioned with other orchestrators.
+
+The Container-Optimized OS does not include a package manager and does not allow non-containerized applications to run. Additionally, its root filesystem is mounted as read-only, which poses a challenge for IO operations.
+
+In GKE, Kubernetes tackes these constraints by housing necessary dependencies in a chroot environment (`/home/kubernetes/containerized_mounter/rootfs`) and mounting directories within it, enabling the execution of required tasks.
+
+Longhorn provides a GKE COS node agent, which leverages GKE Kubernetes solutions to configure and run necessary dependencies. This agent is responsible for the following operations:
```

Comment:
Can you provide more information or links to the official documents for the GKE COS node agent?

Reply:
No. I'm a bit confused when reading `Longhorn provides a GKE COS node agent ...`.  Is "node agent" a tool, defined by GKE platform, used for installation or configuration of the environment in the GKE OS system? Is it provided by Longhorn? Is this term visible in the GKE OS documentation, or is it something we invented ourselves?

Reply:
It's a DaemonsSet Longhorn provides to the user for setting up dependencies and running the iSCSI daemon. Let me add the `DaemonSet` into the sentence to see if it is less confusing.

Reply:
Thanks for your explanation. :)